bgpd/bgp_dump.c

```diff
@@ -494,13 +494,13 @@ static void bgp_dump_common(struct stream *obuf, struct peer *peer,
 }
 
 /* Dump BGP status change. */
-void bgp_dump_state(struct peer *peer, int status_old, int status_new)
+int bgp_dump_state(struct peer *peer)
 {
 	struct stream *obuf;
 
 	/* If dump file pointer is disabled return immediately. */
 	if (bgp_dump_all.fp == NULL)
-		return;
+		return 0;
```

Comment:
why is this conversion needed (void --> int)?

Reply:
function signatures for hook functions are fixed